packages/apputils/src/sessioncontext.tsx

```diff
@@ -198,6 +199,18 @@ export interface ISessionContext extends IObservableDisposable {
    */
   readonly specsManager: KernelSpec.IManager;
 
+  /**
+   * Session dialogs for selecting a kernel.
+   */
+  readonly dialogs: ISessionContext.IDialogs;
```

Comment:
I added dialogs to SessionContext. Without this, bunch of additional code changes are required to inject ISessionContext.IDialogs.

ISessionContext.IDialogs and ISessionContext is closely related, so I think it is okay to defined `dialogs` member inside ISessionContext. However in some perspective like MVC pattern, someone might think it should be avoided.

Reply:
Sorry to be annoying about this one. But indeed I prefer not to provide the `dialogs` from the context.

If this is okay with you, I could push a commit to take care of that.

Reply:
Thats completely OK. I added you as a collaborator to our fork.